oak_containers_orchestrator/src/main.rs

```diff
@@ -68,16 +69,18 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         path.push(IPC_SOCKET_FILE_NAME);
         path
     };
+    let (shutdown_sender, shutdown_receiver) = channel::<()>();
 
     tokio::try_join!(
         oak_containers_orchestrator::ipc_server::create(
             ipc_path,
             encryption_key_provider,
             attester,
             application_config,
-            launcher_client
+            launcher_client,
+            shutdown_receiver
         ),
-        oak_containers_orchestrator::container_runtime::run(&container_bundle)
+        oak_containers_orchestrator::container_runtime::run(&container_bundle, shutdown_sender)
```

Comment:
I am not sure: who actually sends the shutdown?

Reply:
The container runtime sends the notification that the container has exited. The gRPC service then interprest this as a request to shut down.